src/ddprof_cmdline.cc

```diff
@@ -136,11 +141,10 @@ bool watcher_from_tracepoint(const char *_str, PerfWatcher *watcher) {
   // period is a number
   char *str = strdup(_str);
   size_t sz_str = strlen(str);
-  const char *groupname;
-  const char *tracename;
+  const char *gname;
```

Comment:
Don't stare _too_ hard at this `watcher_from_tracepoint()` stuff.  It's all going completely away when #70 is merged.